gym/spaces/graph.py

```diff
@@ -114,12 +51,31 @@ def __init__(
         self.node_space = node_space
         self.edge_space = edge_space
 
-        # graph object creator
-        self.graphObj = GraphObj
-
         super().__init__(None, None, seed)
 
-    def _generate_sample_space(self, base_space: Union[None, Box, Discrete], num: int) -> Optional[Union[Box, Discrete]]:
+    @staticmethod
+    def graph_obj(
+        nodes: np.ndarray, edges: np.ndarray, edge_links: np.array
+    ) -> NamedTuple:
```

Comment:
With the named tuple in the global scope then I don't think this function is needed.